### PR TITLE
Generated SDK Changes for adding orchestrationMode property to a virtual machine scale set

### PR DESCRIPTION
**Pull request by kimberlyyipc**
This is the generated SDK in relation to Rest API PR https://github.com/Azure/azure-rest-api-specs/pull/12062 to expose orchestrationMode property in a virtual machine scale set

### CODE REVIEW

File: sdk/compute/Microsoft.Azure.Management.Compute/tests/VMScaleSetTests/VMScaleSetTestsBase.cs [hunk 1]
```diff
@@ -99,6 +99,33 @@ protected VirtualMachineScaleSetExtension GetAzureDiskEncryptionExtension()
             return vmExtension;
         }
 
+        protected VirtualMachineScaleSet CreateDefaultVMScaleSetInput_Flex(
+            string rgName,
```
Comment (Sandido) [inline sdk/compute/Microsoft.Azure.Management.Compute/tests/VMScaleSetTests/VMScaleSetTestsBase.cs:103]: Why is there no re-recorded json file for this file?

**Review by Sandido — CHANGES_REQUESTED** [review]

File: sdk/compute/Microsoft.Azure.Management.Compute/tests/VMScaleSetTests/VMScaleSetTestsBase.cs (hunk 1 above)
Comment (kimberlyyipc) [reply sdk/compute/Microsoft.Azure.Management.Compute/tests/VMScaleSetTests/VMScaleSetTestsBase.cs:103]: This is not an added test, it is a function I added to create a flex VMSS. I don't see any previous json files associated to this file

**Comment by bquantump** [comment]
Why is this do not merge?

**Comment by bquantump** [comment]
Changed to draft due to no updates

**Comment by kimberlyyipc** [comment]
@bquantump this shouldn't be a draft, this PR is for SDK updates related to this rest api pr: https://github.com/Azure/azure-rest-api-specs/pull/12062

**Comment by bquantump** [comment]
> @bquantump Steve Schilinski FTE this shouldn't be a draft, this PR is for SDK updates related to this rest api pr: [Azure/azure-rest-api-specs#12062](https://github.com/Azure/azure-rest-api-specs/pull/12062) that has already been merged

Why is it [Do-Not-Merge]  in the title then? Why are there merge conflicts? why are is the check enforcer not been debugged in 3 weeks?

**Comment by kimberlyyipc** [comment]
> > @bquantump Steve Schilinski FTE this shouldn't be a draft, this PR is for SDK updates related to this rest api pr: [Azure/azure-rest-api-specs#12062](https://github.com/Azure/azure-rest-api-specs/pull/12062) that has already been merged
> 
> Why is it [Do-Not-Merge] in the title then? Why are there merge conflicts? why are is the check enforcer not been debugged in 3 weeks?

- I've removed [Do-Not-Merge] from the title. It was initially there as we couldn't write tests for a while due to ARM manifest bug. Though I believe the SDK PRs made to the compute-2020-12 branch cannot be merged until the compute-2020-12 rest API is released. @Sandido to confirm.
- I am working on resolving the merge conflicts. Can I directly edit the Generated files with merge conflicts? Or do I have to edit my swagger branch and regenerate the SDK? 
- I was not aware of this check enforced bug. What is wrong with it?

**Comment by bquantump** [comment]
> > > @bquantump
> > >             
> > >             Steve Schilinski
> > >             FTE Steve Schilinski FTE this shouldn't be a draft, this PR is for SDK updates related to this rest api pr: [Azure/azure-rest-api-specs#12062](https://github.com/Azure/azure-rest-api-specs/pull/12062) that has already been merged
> > 
> > 
> > Why is it [Do-Not-Merge] in the title then? Why are there merge conflicts? why are is the check enforcer not been debugged in 3 weeks?
> 
> * I've removed [Do-Not-Merge] from the title. It was initially there as we couldn't write tests for a while due to ARM manifest bug. Though I believe the SDK PRs made to the compute-2020-12 branch cannot be merged until the compute-2020-12 rest API is released. @Sandido
>               
>               Adam Sandor
>               FTE to confirm.
> * I am working on resolving the merge conflicts. Can I directly edit the Generated files with merge conflicts? Or do I have to edit my swagger branch and regenerate the SDK?
> * I was not aware of this check enforced bug. What is wrong with it?

You cannot manually edit generated code, the CI will force a failure if it detects that. 

This is a large print out above on how to debug the check enforcer.

**Comment by Sandido** [comment]
closing for now.

**Comment by Sandido** [comment]
We will revisit this PR when it is ready to be reviewed. 
WE do not expect the PR owner to debug the check enforcer.

**Pull request closed without merging**